lib/internal/test_runner/runner.js

```diff
@@ -100,6 +100,11 @@ function createTestFileList(patterns) {
   });
   const results = glob.globSync();
 
+  // We need to resolve paths as this function's cwd may differ from the process cwd
+  ArrayPrototypeForEach(results, (result, i) => {
+    results[i] = resolve(cwd, result);
+  });
```

Comment:
I think you've reinvented `ArrayPrototypeMap` 😄

Reply:
At first, it was a for loop 😬
I'm going to replace it with a map while removing the comment

Reply:
done @cjihrig